CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+1.0.3 / 2021-08-17
+==================
+- Fixed `EventEmitter` memory leak (see issue [#81](https://github.com/apify/proxy-chain/issues/81))
```

Comment:
btw you can use just the `#81`, gh will do the link for you automatically if the issue/pr is in the same repository